modules/models/dynamic/single_track.hpp

```diff
@@ -16,22 +16,22 @@ namespace modules {
 namespace models {
 namespace dynamic {
 
-
 class SingleTrackModel : public DynamicModel {
  public:
   explicit SingleTrackModel(const modules::commons::ParamsPtr& params)
       : DynamicModel(params),
-        wheel_base_(2.7),
-        steering_angle_max_(0.2),
-        lat_acceleration_max_(4.0) {
-    wheel_base_ = params->GetReal("DynamicModel::wheel_base",
-                                  "Wheel base of vehicle [m]", 2.7);
-    steering_angle_max_ = params->GetReal("DynamicModel::delta_max",
-                                          "Maximum Steering Angle [rad]", 0.2);
-    lat_acceleration_max_ =
-        params->GetReal("DynamicModel::lat_acc_max",
-                        "Maximum lateral acceleration [m/s^2]", 4.0);
-  }
+        wheel_base_(params->GetReal("DynamicModel::wheel_base",
+                                    "Wheel base of vehicle [m]", 2.7)),
+        steering_angle_max_(params->GetReal(
+            "DynamicModel::delta_max", "Maximum Steering Angle [rad]", 0.2)),
+        lat_acceleration_max_(
+            params->GetReal("DynamicModel::lat_acc_max",
+                            "Maximum lateral acceleration [m/s^2]", 4.0)),
+        max_acceleration_(params->GetReal("DynamicModel::max_acceleration",
```

Comment:
In accordance with lat_acceleration_max_, we should name this long_acceleration_max_ (similar with _min). Also consider removing the dynamic / static description.